lib/mongo/client_encryption.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (C) 2019 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  # ClientEncryption encapsulates explicit operations on a key vault
+  # collection that cannot be done directly on a MongoClient. It
+  # provides an API for explicitly encrypting and decrypting values,
+  # and creating data keys.
+  class ClientEncryption
+
+    # Create a new ClientEncryption object with the provided options.
+    #
+    # @param [ Mongo::Client ] client A Mongo::Client
+    #   that is connected to the MongoDB instance where the key vault
+    #   collection is stored.
+    # @param [ Hash ] options The ClientEncryption options
+    #
+    # @option options [ String ] key_vault_namespace The name of the
```

Comment:
We use `:key_vault_namespace` etc. to indicate that options must be passed as symbols.

Reply:
Thanks!